docs: mention xdebug_ide_location setting in xdebug troubleshooting section (#5307)
ddev/ddev: docs/content/users/debugging-profiling/step-debugging.md

* Reboot your computer.
* If you're running WSL2 and have PhpStorm running inside WSL2 (the Linux version of PhpStorm) then `ddev config global --xdebug-ide-location=wsl2`.
* Temporarily disable any *firewall* or *VPN* if you’re having trouble. Xdebug is a network protocol, and the PHP process inside the web container must be able to establish a TCP connection to the listening IDE (PhpStorm, for example).

> * Confirm that DDEV’s [`xdebug_ide_location`](../../users/configuration/config/#xdebug_ide_location) config setting is set properly, which in most cases should be set to an empty string. Check both your project's `.ddev/config.yaml` and DDEV’s global `~/.ddev/global_config.yaml`.

Reviewer: Pretty sure this should be: (anyway, failed link check)

```suggestion
* Confirm that DDEV’s [`xdebug_ide_location`](../../configuration/config/#xdebug_ide_location) config setting is set properly, which in most cases should be set to an empty string. Check both your project's `.ddev/config.yaml` and DDEV’s global `~/.ddev/global_config.yaml`.
```

Reviewer 2: Wow, I checked it locally so it’s kind of amazing I still managed to get it wrong. Thanks for fixing!

Reviewer: It took me three tries. Then go back to the IDE locally, where it helped me get the right syntax. It would help if the markdown syntax were the same as http syntax.

> * Check to confirm that the DDEV configuration parameter `xdebug-ide-location` is set properly. For most situations, this should be set to an empty string. Be sure to check both your project's .ddev/config.yaml and ~/.ddev/global_config.yaml. More information about this parameter can be found in ~/.ddev/global_config.yaml. 

Reviewer 2: It looks like the config setting should use underscores and that there isn’t an equivalent [config command](https://ddev.readthedocs.io/en/stable/users/usage/commands/#config).

I’d edit this to use underscores, link to the relevant documentation instead of mentioning the comments in the global config file, wrap filenames in backticks for consistency, gently reiterate project YAML vs. global YAML, and whittle things down for brevity:

```suggestion
* Confirm that DDEV’s [`xdebug_ide_location`](../../users/configuration/config/#xdebug_ide_location) config setting is set properly, which in most cases should be set to an empty string. Check both your project's `.ddev/config.yaml` and DDEV’s global `~/.ddev/global_config.yaml`.
```

Author: Yep, works for me.

Reviewer 2: You can click **Commit suggestion** if you’re cool with it.
